packages/upstash-redis/tests/index.test.ts

```diff
@@ -0,0 +1,39 @@
+import upstashRedisClient from "@upstash/redis"
+import { runBasicTests } from "../../../basic-tests"
+import UpstashRedisAdapter from "../src"
+
+const client = upstashRedisClient(
+  process.env.NEXTAUTH_UPSTASH_REDIS_URL,
+  process.env.NEXTAUTH_UPSTASH_REDIS_KEY
```

Comment:
```suggestion
  process.env.UPSTASH_REDIS_URL,
  process.env.UPSTASH_REDIS_KEY
```

I feel this prefix is unnecessary in this context.